src/main/kotlin/org/jmailen/gradle/kotlinter/tasks/GitHookTasks.kt

```diff
@@ -33,8 +36,18 @@ open class InstallPrePushHookTask : InstallHookTask("pre-push") {
  * Install or update a kotlinter-gradle hook.
  */
 abstract class InstallHookTask(@get:Internal val hookFileName: String) : DefaultTask() {
+
     @Input
-    val gitDirPath: Property<String> = project.objects.property(default = ".git")
+    val gitDirPath = project.objects.property(default = ".git")
+
+    @get:InputDirectory
```

Comment:
🐛 ?
`@InputDirectory` documentation states:
>This will cause the task to be considered out-of-date when the directory location **or contents** have changed.

It doesn't sound valid to listen for contents changes of the whole `projectDirectory` - I wouldn't expect the task to be re-run if any file within the project changes 👀